packages/@pollyjs/persister/src/har/response.js

```diff
@@ -37,6 +37,10 @@ export default class Response {
 
     if (response.body && typeof response.body === 'string') {
       this.content.text = response.body;
+
+      if (response.isBinary) {
+        this.content._isBinary = true;
```

Comment:
`_isBinary` is only going to the written to the har content if the content is binary. I didnt think it was necessary to always include it. @jasonmit thoughts?

Reply:
seems fair but some what on topic - does this constitute a breaking change?

Reply:
No, this is backwards compatible but does require to re-record to resolve the issue.